### PR TITLE
Throw specific error if no response body or header is present

### PR DESCRIPTION
**Pull request by hpurmann**
This PR makes the error message more user friendly in case he tries to
validate a response without body or header.
Additionally, it renames the variable to reflect that we are getting a
validator as in the other cases.

See https://github.com/Zooz/api-schema-builder/tree/5d6ff44ecc298b7cbd4804b6f19eba5950a42793#response

### CODE REVIEW

File: src/validate-json-schema-middleware.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import get from 'lodash/get'
 
 const getBodyValidator = swaggerEndPoint => swaggerEndPoint.body
 const getParametersValidator = swaggerEndPoint => swaggerEndPoint.parameters
-const getResponseEndPoint = swaggerEndPoint => swaggerEndPoint.responses
+const getResponseValidator = swaggerEndPoint => swaggerEndPoint.responses
```
Comment (pithu) [inline src/validate-json-schema-middleware.js]: getResponseValidator is for me a valid validator object with a validate function and an error object.
getResponseEndPoint is the correct name here, why did you rename it ?

File: src/validate-json-schema-middleware.js [hunk 2]
```diff
@@ -64,10 +64,13 @@ const OpenApiValidator = async (swaggerPath, options) => {
 
   const getResponseValidationEndPoint = (path) => {
     const swaggerEndPoint = getSwaggerEndPoint(schema, path)
-    const responseEndPoint = getResponseEndPoint(swaggerEndPoint)
+    const responseValidators = getResponseValidator(swaggerEndPoint)
     return {
       validator: (httpStatusCode) => {
-        const validator = responseEndPoint[httpStatusCode]
+        const validator = responseValidators[httpStatusCode]
+        if (validator == null) {
```
Comment (pithu) [inline src/validate-json-schema-middleware.js]: In all other places i did a simple `if (!validator)` check, i would prefer to have it consistent.

**Comment by hpurmann** [comment]
Yes, that's great, as we have discussed. Thank you!
